### ISSUE TITLE
onTimeElapsed

### ISSUE THREAD

**Issue by gsnadav** [haxball/haxball-issues#309]
An event which is called when the time on the clock is elapsed. Currently only onGameTick is available, but onGameTick is called also when the clock is stopped (after scoring a goal and before touching the ball once again) so there's no coherent way to do something on time.

**Comment by basro** [comment]
You can do this using the getScores() function which returns a struct with time and timeLimit in it.
Here's an example: https://github.com/haxball/haxball-issues/blob/master/headless/examples/gameTime.js

**Comment by gsnadav** [comment]
Yes, but would you recommend to call it on every tick? Isn't it a little expensive?

**Comment by Leo-MapCreator** [comment]
@basro I have a doubt about what gsnadava said, about expensiveness, where do i ask you about this? Create a new topic or in this topic, even though this topic is about another thing?

**Comment by Leo-MapCreator** [comment]
@basro I have an variable in the angle of the uniform, and it changes per kick, making it a rotating uniform, is this expensive? Cause sometimes I feel the ping rises when this is enabled
